### ISSUE TITLE
More detailed MENUDEF

### ISSUE THREAD

**Issue by Shockwave508** [BLOODWOLF333/Brutal-Doom-Community-Expansion#40]
### Description
Some options are poorly-explained in Brutal Doom's menus; adding text blurbs beneath each toggle could assist certain people with setting the game up the way they want to.

**Comment by Shockwave508** [comment]
Here's an updated MENUDEF that I put together. [MENUDEF.TXT](https://github.com/BLOODWOLF333/Brutal-Doom-Community-Expansion/files/4783113/MENUDEF.TXT)

**Comment by BLOODWOLF333** [comment]
Added in commit 1e14c01

**Comment by BLOODWOLF333** [comment]
The BD Level options are way too wordy.  At 1920x1080 with text scaling on, the wall of text it cut off below the demon spheres option.  It's description is not shown so, this needs to be trimmed down to preferably single-line explanations

**Comment by BLOODWOLF333** [comment]
....I didnt know you could scroll down _head desk_ derp

**Comment by BLOODWOLF333** [comment]
Added in commit bcd04bb
